common/web/keyboard-processor/src/text/codes.ts

```diff
@@ -1,34 +1,25 @@
-// TODO:  Move to separate folder:  'codes'
-// We should start splitting off code needed by keyboards even without a KeyboardProcessor active.
-
-// see also: common/web/types/src/kmx/kmx.ts
+import { ModifierKeyConstants } from '@keymanapp/common-types';
 
 const Codes = {
-  // Define Keyman Developer modifier bit-flags (exposed for use by other modules)
-  // Compare against /common/include/kmx_file.h.  CTRL+F "#define LCTRLFLAG" to find the secton.
   modifierCodes: {
```

Comment:
```suggestion
  // Debug-mode keyboards compiled before Keyman 18.0 referenced the `ModifierKeyConstants` 
  // constants via the names established below.  We must continue to support them, as they're
  // essentially part of the keyboard API now.
  modifierCodes: {
```

----

Optionally, we could also emit the same values under their new name:

```suggestion
  // Debug-mode keyboards compiled before Keyman 18.0 referenced the `ModifierKeyConstants` 
  // constants via the names established below.  We must continue to support them, as they're
  // essentially part of the keyboard API now.
  modifierCodes: {
    ...ModifierKeyConstants,
```

so that at some point in the distant future, we _could_ drop the old names entirely... maybe.